### PR TITLE
conv2d PyTorch Frontend

### PR DESCRIPTION
**Pull request by Leonard-Zeng**
- [ ] #4041

### CODE REVIEW

File: ivy_tests/test_ivy/test_frontends/test_torch/test_convolution_functions.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# global
```
Comment (Unevilicorn) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_convolution_functions.py:1]: some lint errors in this function

File: ivy/functional/frontends/torch/convolution_functions.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import ivy
+
+def conv2d(input, weight, bias=None, stride=1, padding=0, dilation=1, groups=1):
```
Comment (Unevilicorn) [inline ivy/functional/frontends/torch/convolution_functions.py:3]: lint errors in this function

**Comment by Unevilicorn** [comment]
The changes looks good! and no more error is showing up for your tests. However the `vcs.xml` is still showing up as being changed, when this is solved, it will be good for merging.

**Comment by Leonard-Zeng** [comment]
> The changes looks good! and no more error is showing up for your tests. However the `vcs.xml` is still showing up as being changed, when this is solved, it will be good for merging.

All done!

**Comment by Unevilicorn** [comment]
Perfect, I'll get this merged :+1:

**Pull request merged**